.github/workflows/tests-datapath-verifier.yaml

```
@@ -100,6 +100,9 @@ jobs:
             ci-kernel: '61'
           # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
           - kernel: '6.6-20240710.064909'
+            ci-kernel: '61'
+          # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
+          - kernel: 'bpf-next-20240711.013133'
```

To preserve the current branch content, this should be:

```suggestion
          - kernel: 'kernel: '6.6-20240710.064909''
```

(in stable branches, we replaced the bpf-next kernel with a stable kernel)

---

@julianwiedmann cool catch! thank you!!!
Shouldn't I remove the following lines at all:
```
          # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
          - kernel: 'bpf-next-20240711.013133'
            ci-kernel: 'netnext'
```
?

If I replace `'bpf-next-20240711.013133'` with `'6.6-20240710.064909'` it looks like the previous kernel version...

---

I'd expect that when backporting https://github.com/cilium/cilium/pull/34420 and inserting the diff in the right spot, we end up with the following:

``` 
          # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
          - kernel: '6.1-20240710.064909'
            ci-kernel: '61'
          # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
          - kernel: '6.6-20240710.064909'
            ci-kernel: '61'
          # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
          - kernel: '6.6-20240710.064909'
            ci-kernel: 'netnext'
```

Which is three distinct test configs (the `ci-kernel` parameter selects what datapath configs should be tested on-top of the specified kernel)

---

Gotcha, fixed! Thank you!

---

looks good now, thank you!